src/Mapping/Operator/Factory/InputQuantityValue.php

```diff
@@ -24,11 +24,11 @@ class InputQuantityValue extends QuantityValue
      * @param mixed $inputData
      * @param bool $dryRun
      *
-     * @return \Pimcore\Model\DataObject\Data\QuantityValue
+     * @return \Pimcore\Model\DataObject\Data\InputQuantityValue
      */
     public function process($inputData, bool $dryRun = false)
     {
-        return new \Pimcore\Model\DataObject\Data\QuantityValue(
+        return new \Pimcore\Model\DataObject\Data\InputQuantityValue(
```

Comment:
without this commit, the import process fails 
![image](https://github.com/pimcore/data-importer/assets/6014195/2204ca6c-86a8-44d0-a64a-e503059f8bd0)